builds/azure-pipelines/template-steps-build-test.yml

```diff
@@ -56,7 +57,7 @@ steps:
   inputs:
     command: build
     projects: '${{ parameters.solution }}'
-    arguments: '--configuration ${{ parameters.configuration }} -p:GeneratePackageOnBuild=false'
+    arguments: '--configuration ${{ parameters.configuration }} -p:GeneratePackageOnBuild=false -p:Version=${{ parameters.binariesVersion }}'
```

Comment:
You seem to have some commits that are already in main. Please make sure you've rebased and merged in commits from main correctly so they don't show up.